container/build.py

```diff
@@ -27,38 +30,41 @@
 will produce docker locally as {container_type}:latest
 
 usage:
-  build.py -d {rbe-debian8,rbe-debian9,rbe-ubuntu16_04,ubuntu16_04-bazel-docker,ubuntu16_04-bazel}
-      [-p PROJECT] [-c CONTAINER] [-t TAG] [-a] [-b BUCKET] [-h]
-      [-l]
+  build.py [-d TYPE] [-p PROJECT] [-c CONTAINER] [-t TAG] [-a] 
+           [-b BUCKET] [-h] [-m MAP] [-l]
 
 required arguments:
-  -d TYPE, --type TYPE  Type of the container: see TYPE_TARGET_MAP
+  -d TYPE, --type TYPE  Type of the container: see SUPPORTED_TYPES
   -p PROJECT, --project PROJECT
                         GCP project ID
   -c CONTAINER, --container CONTAINER
                         Docker container name
   -t TAG, --tag TAG     Docker tag for the image
+  -v BAZEL_VERSION, --bazel_version BAZEL_VERSION
+                        The version of Bazel to build the image with, e.g. 0.15.1
 
 optional arguments:
   -a, --async           Asynchronous execute Cloud Container Builder
   -b BUCKET, --bucket BUCKET
                         GCS bucket to store the tarball of debian packages
   -h, --help            print this help text and exit
+  -m MAP, --map MAP     overrides target map file path
```

Comment:
can you explain what the effect of overriding it is (or when it needs to be overriden?)

Reply:
Added a better explanation

Reply:
you should be clear that it's taking a path, and will override TYPE_PACKAGE_MAP, TYPE_TARGET_MAP, TYPE_TARBALL_MAP